plugins/postcss-base-plugin/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "@csstools/postcss-base-plugin",
+  "private": "true",
+  "version": "0.0.0",
+  "description": "A base plugin",
+  "author": "Jonathan Neal <jonathantneal@hotmail.com>",
+  "license": "CC0-1.0",
+  "engines": {
+    "node": "^12 || ^14 || >=16"
+  },
+  "main": "dist/index.cjs",
+  "module": "dist/index.mjs",
+  "types": "dist/index.d.ts",
+  "files": [
+    "CHANGELOG.md",
+    "INSTALL.md",
+    "LICENSE.md",
+    "README.md",
+    "dist"
+  ],
+  "bin": {
+    "postcss-base-plugin": "dist/cli.mjs"
+  },
+  "scripts": {
+    "prepublishOnly": "npm run build --if-present && npm run test --if-present",
+    "lint": "eslint src/**/*.ts",
+    "test": "postcss-tape && bash ./test/cli/test.sh",
+    "build": "rollup -c ../../rollup/default.ts.js",
+    "stryker": "stryker run --logLevel error"
+  },
+  "devDependencies": {
+    "@csstools/base-cli": "^0.1.0",
+    "postcss": "^8.3.6",
+    "postcss-tape": "^6.0.1"
+  },
+  "peerDependencies": {
+    "postcss": "^8.3"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/csstools/postcss-plugins.git",
+    "directory": "plugins/postcss-base-plugin"
+  }
+}
```

Comment:
In `postcss-nesting` there is an `exports` field.
Do we need or want this going forward?

https://github.com/csstools/postcss-nesting/blob/main/package.json#L18-L24

Reply:
I don't think we do need that given that's it's pretty limited what we send to `npm` when packed and published. Do you want to keep it?